### DISCUSSION TITLE
gitlab: renovate/artifacts external job error?

### DISCUSSION THREAD

**Discussion by strowi** [renovatebot/renovate#10714] [Request Help]
Hi everyone,

i just stumpled upon this an couldn't find any references to this:
![grafik](https://user-images.githubusercontent.com/589662/124488182-d9525900-ddaf-11eb-9fc9-935fe7abe22d.png)

Where does this external job come from (we have no such configuration anywhere) ? It is a dead link, so no logs or job to click through..
I speculate this might be related to the "build"-job which fails due to some yarn errors?

![grafik](https://user-images.githubusercontent.com/589662/124488477-2e8e6a80-ddb0-11eb-8073-9df42c8f6ccb.png)

Could anyone clarify where this external job comes from ?

regards,
strowi

**Comment by viceice** [comment]
You have renovate running, you should check your renovate runner logs for the cause.

**Reply by strowi** [reply]
Turns out the error is a 404 on some package, which gets reported correctly.
I was just wondering about the "external job" appearing with a renovate/artifacts job, that hasn't been configured anywhere.
We are running renovate as a cronjob.
(sorry forgot to mention that in the first post).

**Reply by viceice** [reply]
the `renovate/artifacts` is a commit status, which renovate posts to signal a update failure when calling npm / yarn / ...

So this is expected and works as intended.

**Reply by strowi** [reply]
thanks @viceice for the explanation. Thats what i was looking for.;)
